js/indicators/indicators.src.js

```diff
@@ -112,6 +112,16 @@ seriesType(
          * @type {string}
          */
         linkedTo: undefined,
+        /**
+         * Whether to compare indicator to the main series values
+         * or indicator values.
+         *
+         * @sample {highstock} stock/plotoptions/series-comparetomain/
+         *         compareToMain
```

Comment:
Could you change the name of the demo to be more user friendly? @raf18seb Also, it would be a good idea to provide a demo that shows two different settings at the same time to highlight the difference.

Reply:
Sure, I'll do it soon.